addons/project/data/project_demo.xml

```diff
@@ -758,6 +758,30 @@
             <field name="body"><![CDATA[<ul class="mb-0 list-unstyled d-flex flex-column gap-1"><li class="o-mail-Message-tracking" role="group"><span class="o-mail-Message-trackingOld text-muted fw-bold">In Progress</span><i class="o-mail-Message-trackingSeparator fa fa-long-arrow-right mx-2 text-600"></i><span class="o-mail-Message-trackingNew me-1 fw-bold text-info">Done</span><span class="o-mail-Message-trackingField ms-1 fst-italic text-muted">(Stage)</span></li></ul>]]></field>
         </record>
 
+        <record id="project.project_1_task_2" model="project.task">
+            <field name="duration_tracking"
+                eval="{
+                    'd': DateTime.now() - relativedelta(months=1),
+                    's': ref('project.project_stage_2'),
+                    ref('project.project_stage_0'): ((DateTime.now() - relativedelta(months=2)) - (DateTime.now() - relativedelta(months=5))).total_seconds(),
```

Comment:
why not replacing` ((DateTime.now() - relativedelta(months=2)) - (DateTime.now() - relativedelta(months=5))).total_seconds()`
by just` 3*30*24*3600`